openpype/pipeline/colorspace.py

```diff
@@ -748,10 +748,12 @@ def get_views_data_subprocess(config_path):
 def get_imageio_config(
     project_name,
     host_name,
+    asset_name=None,
+    task_name=None,
     project_settings=None,
-    anatomy_data=None,
     anatomy=None,
-    env=None
+    env=None,
+    anatomy_data=None,
```

Comment:
Why are you swapping these arguments around? Best not to do that unless you have very good reasons to do so.

Reply:
To be honest, this change had been suggested by @iLLiCiTiT and I had already forgotten the reason. Perhaps he will still remember the argument for this?

Reply:
I don't remember, but would rename it to `template_data`.